src/node.cc

```diff
@@ -3535,6 +3547,7 @@ static void PrintHelp() {
          "  --trace-deprecation        show stack traces on deprecations\n"
          "  --throw-deprecation        throw an exception on deprecations\n"
          "  --no-warnings              silence all process warnings\n"
+         "  --napi-modules[=yes|no]    load N-API modules (default no)\n"
```

Comment:
@Fishrock123 can you explain why you (or someone) might want to disable the feature after it is out of experimental status?